src/helpers/worker.js

```diff
@@ -57,12 +62,17 @@ function transformConfig(config) {
  */
 function jitMiddleware(config) {
   const cfg = transformConfig(config);
+  const tmpPath = homePath('cache/jit', cfg.hash);
```

Comment:
does it make sense to split `'cache/jit'` into `'cache'` and `'jit'`, so it's OS agnostic?

Reply:
Node.js resolves it across OS, so there is not difference